freqtrade/exchange/__init__.py

```diff
@@ -286,6 +318,23 @@ def get_ticker(self, pair: str, refresh: Optional[bool] = True) -> dict:
             logger.info("returning cached ticker-data for %s", pair)
             return self._cached_ticker[pair]
 
+    async def get_ticker_history_async(self, tick_interval: str,
+                                       since_ms: Optional[int] = None):
```

Comment:
where do you use since_ms ?

Reply:
it's part of the original call .... so it's safe to assume that it was copied for simplicity